_docs/layout/navigation.md

```diff
@@ -7,33 +7,42 @@ preview-image: layout/preview-images/navigation.svg
 resource: true
 ---
 
-Navigation establishes where the user is in an application and provides a path
-to complete tasks. Consistent navigation patterns allow the user to complete
-tasks more efficiently and confidently. Rackspace applications may utilize a
-combination of eyebrow navigation, either a top or side navigation, and
-breadcrumbs.
+Navigation establishes a user's location within an application and
+shows the path forward to accomplish a goal. Consistent navigation
+patterns increase user confidence and efficiency. You can use a
+combination of eyebrow navigation, either top or side navigation, and
+breadcrumbs to guide users.
 
 {% include toc.html %}
 
-# Primary Navigation {% include complete.html %}
+# Primary navigation {% include complete.html %}
 
-The primary navigation is used for the main sections of the application or for
-dropdown categories that organize sections into higher level groups.
+Use primary navigation for the main sections of an application or for
+drop-down categories that organize sections into higher-level groups.
 
-## Left Navigation
+## Left navigation
 
 <div class="row">
 {% column left:"col-md-4" %}
-The Left Navigation component is used on all web applications that desire to
-implement a highly compliant version of the Helix Design Language.
 
--   Can extend to fit any navigation IA via scrolling when the list of
-    navigational items exceeds the screen view-height.
+Use the left navigation component for web applications when you want
+to implement a highly-compliant version of the Helix design language.
```

Comment:
Since we are moving all applications to using a side navigation, maybe change the sentence to:
Use the left navigation component for you web application unless there is a firm implementation constraint that require you to use a top navigation.